srcpkgs/limine/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'limine'
+pkgname=limine
+version=3.0.3
+revision=1
+archs="x86_64 x86_64-musl"
+#wrksrc=
```

Comment:
Half of the variables in here are commented out, the commented ones should be removed just like in other templates.